src/Assert.php

```diff
@@ -852,6 +852,39 @@ public static function inArray(mixed $value, mixed $values, string $message = ''
         }
     }
 
+    /**
+     * A more human-readable alias of Assert::notInArray().
+     *
+     * @psalm-pure
+     *
+     * @throws InvalidArgumentException
+     */
+    public static function notOneOf(mixed $value, mixed $values, string $message = ''): void
+    {
+        static::notInArray($value, $values, $message);
+    }
+
+    /**
+     * Does strict comparison, so Assert::notInArray(3, [1, 2, 3]) will not pass
+     * the assertion, but Assert::notInArray(3, ['3']) will.
```

Comment:
This would be the description, so the bloc needs a title like:

```suggestion
     * Check that a value is not present
     *
     * Does strict comparison, so Assert::notInArray(3, [1, 2, 3]) will not pass
     * the assertion, but Assert::notInArray(3, ['3']) will.
```